tests/Integration/Fixtures/redirects.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+function redirect(int $status, string $location): void
```

Comment:
Can we use anonymous function instead of populating the global namespace?
```suggestion
$redirect = function(int $status, string $location): void
```

Reply:
Good point. I have moved it to a separate file as per PSR-1:

> Files SHOULD either declare symbols (classes, functions, constants, etc.) or cause side-effects (e.g. generate output, change .ini settings, etc.) but SHOULD NOT do both.